package.json

```diff
@@ -201,7 +199,8 @@
   "parcel-namer-rewrite": {
     "developmentHashing": true,
     "rules": {
-      "(.*).png": "$1.png"
+      "(.*).png": "$1.png",
+      "(.*).woff(.*)": "$1.woff$2"
```

Comment:
this looks expensive and too broad (component.woffles.js)

```json
"(.*).woff": "$1.woff",
"(.*).woff2": "$1.woff2"
```

but this will probably go away after combining with https://github.com/oasisprotocol/oasis-wallet-web/pull/778/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L218

Reply:
I will port removal of  `parcel-namer-rewrite`. Extension side branch is still in experimental state.